tests/Unit/Service/ResponseBuilderTest.php

```diff
@@ -43,12 +44,7 @@ public function testBuildResponseWithJsonSerializable()
         $builder = new ResponseBuilder();
         $this->expectException(RuntimeException::class);
         $this->expectExceptionMessage('expected');
-        $builder->buildResponse(new class() implements JsonSerializable {
-            public function jsonSerialize()
```

Comment:
I suppose we use the [`#[\ReturnTypeWillChange`](https://php.watch/versions/8.1/ReturnTypeWillChange) attribute instead.